[     UTC     ] Logs for ann-abandono-clientes-hz6sxhgdblx6nxwukfehid.streamlit.app/

────────────────────────────────────────────────────────────────────────────────────────

[03:11:35] 🖥 Provisioning machine...

[03:11:35] 🎛 Preparing system...

[03:11:35] ⛓ Spinning up manager process...

[03:11:37] 🚀 Starting up repository: 'ann-abandono-clientes', branch: 'main', main module: 'app.py'

[03:11:37] 🐙 Cloning repository...

[03:11:37] 🐙 Cloning into '/mount/src/ann-abandono-clientes'...

[03:11:37] 🐙 Cloned repository!

[03:11:37] 🐙 Pulling code changes from Github...

[03:11:38] 📦 Processing dependencies...


──────────────────────────────────────── uv ───────────────────────────────────────────


Using uv pip install.

Using Python 3.12.10 environment at /home/adminuser/venv

Resolved 63 packages in 542ms

Prepared 63 packages in 11.73s

Installed 63 packages in 943ms

 + absl-py==2.2.2

 + altair==5.5.0

 + astunparse==1.6.3

 + attrs==25.3.0

 + blinker==1.9.0

 + cachetools==5.5.2

 + certifi==2025.4.26

 + charset-normalizer==3.4.2

 + click==8.2.0

 + flatbuffers==25.2.10[2025-05-13 03:11:51.880462] 

 + gast==0.6.0

 + gitdb==4.0.12

 + gitpython==3.1.44

 + google-pasta==0.2.0

 + grpcio[2025-05-13 03:11:51.880692] ==1.71.0

 + h5py==3.13.0

 + idna==3.10

 + jinja2==3.1.6

 + jsonschema==4.23.0

 + jsonschema-specifications==2025.4.1

 + keras==3.9.2

 + libclang==18.1.1

 + markdown==3.8

 + markdown-it-py==3.0.0

 + markupsafe==3.0.2

 + mdurl==0.1.2

 + ml-dtypes==0.5.1

 + namex==0.0.9

 [2025-05-13 03:11:51.881074] + narwhals==1.39.0

 + numpy==2.1.3

 + opt-einsum==3.4.0

 + optree==0.15.0

 + packaging==24.2

 + pandas==2.2.3[2025-05-13 03:11:51.881197] 

 + pillow==11.2.1

 + protobuf==5.29.4

 + pyarrow==20.0.0

 + pydeck==0.9.1

 + pygments==2.19.1

 + [2025-05-13 03:11:51.881301] python-dateutil==2.9.0.post0

 + pytz==2025.2

 + referencing==0.36.2

 + requests==2.32.3

 + rich[2025-05-13 03:11:51.881401] ==14.0.0

 + rpds-py==0.24.0

 + setuptools==80.4.0

 + six==1.17.0

 + smmap==5.0.2

 + streamlit==1.45.1

 + tenacity[2025-05-13 03:11:51.881557] ==9.1.2

 + tensorboard==2.19.0

 + tensorboard-data-server==0.7.2

 + tensorflow==2.19.0

 + termcolor==3.1.0

 + toml==0.10.2

 + tornado==6.4.2

 +[2025-05-13 03:11:51.881671]  typing-extensions==4.13.2

 + tzdata==2025.2

 + urllib3==2.4.0

 + watchdog==6.0.0

 + werkzeug==3.1.3

 + wheel==0.45.1

 + wrapt==1.17.2

Checking if Streamlit is installed

Found Streamlit version 1.45.1 in the environment

Installing rich for an improved exception logging

Using uv pip install.

Using Python 3.12.10 environment at /home/adminuser/venv

Audited 1 package in 2ms


────────────────────────────────────────────────────────────────────────────────────────


[03:11:54] 🐍 Python dependencies were installed from /mount/src/ann-abandono-clientes/requirements.txt using uv.

Check if streamlit is installed

Streamlit is already installed

[03:11:55] 📦 Processed dependencies!




2025-05-13 03:12:03.013968: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.

2025-05-13 03:12:03.018620: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.

2025-05-13 03:12:03.027769: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:467] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered

WARNING: All log messages before absl::InitializeLog() is called are written to STDERR

E0000 00:00:1747105923.052217     196 cuda_dnn.cc:8579] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered

E0000 00:00:1747105923.059932     196 cuda_blas.cc:1407] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered

W0000 00:00:1747105923.082947     196 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

W0000 00:00:1747105923.082995     196 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

W0000 00:00:1747105923.082998     196 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

W0000 00:00:1747105923.083000     196 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

2025-05-13 03:12:03.090231: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.

To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.

────────────────────── Traceback (most recent call last) ───────────────────────

  /home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/scriptru  

  nner/exec_code.py:121 in exec_func_with_error_handling                        

                                                                                

  /home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/scriptru  

  nner/script_runner.py:645 in code_to_exec                                     

                                                                                

  /mount/src/ann-abandono-clientes/app.py:4 in <module>                         

                                                                                

      1 import streamlit as st                                                  

      2 import numpy as np                                                      

      3 import tensorflow as tf                                                 

  ❱   4 from sklearn.preprocessing import StandardScaler, LabelEncoder, OneHot  

      5 import pandas as pd                                                     

      6 import pickle                                                           

      7 from pathlib import Path                                                

────────────────────────────────────────────────────────────────────────────────

ModuleNotFoundError: No module named 'sklearn'

────────────────────── Traceback (most recent call last) ───────────────────────

  /home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/scriptru  

  nner/exec_code.py:121 in exec_func_with_error_handling                        

                                                                                

  /home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/scriptru  

  nner/script_runner.py:645 in code_to_exec                                     

                                                                                

  /mount/src/ann-abandono-clientes/app.py:4 in <module>                         

                                                                                

      1 import streamlit as st                                                  

      2 import numpy as np                                                      

      3 import tensorflow as tf                                                 

  ❱   4 from sklearn.preprocessing import StandardScaler, LabelEncoder, OneHot  

      5 import pandas as pd                                                     

      6 import pickle                                                           

      7 from pathlib import Path                                                

────────────────────────────────────────────────────────────────────────────────

ModuleNotFoundError: No module named 'sklearn'

[03:14:08] 🐙 Pulling code changes from Github...

[03:14:09] 📦 Processing dependencies...


──────────────────────────────────────── uv ───────────────────────────────────────────


Using uv pip install.

Using Python 3.12.10 environment at /home/adminuser/venv

Resolved 68 packages in 968ms

Prepared 5 packages in 1.03s

Installed 5 packages in 57ms

 + joblib==1.5.0

 +[2025-05-13 03:14:11.556484]  pickle-mixin==1.0.2

 + scikit-learn==1.6.1

 + scipy==1.15.3

 + threadpoolctl==3.6.0

Checking if Streamlit is installed

Found Streamlit version 1.45.1 in the environment

Installing rich for an improved exception logging

Using uv pip install.

Using Python 3.12.10 environment at /home/adminuser/venv

Audited 1 package in 2ms


────────────────────────────────────────────────────────────────────────────────────────


[03:14:13] 🐍 Python dependencies were installed from /mount/src/ann-abandono-clientes/requirements.txt using uv.

[03:14:13] 📦 Processed dependencies!

  Stopping...




[03:14:17] 🔄 Updated app!

[03:14:45] 🐙 Pulling code changes from Github...

2025-05-13 03:14:45.469 Received event for non-watched file: /mount/src/ann-abandono-clientes/app.py

[03:14:45] 📦 Processing dependencies...

[03:14:45] 📦 Processed dependencies!

[03:14:47] 🔄 Updated app!

[03:14:54] 🔌 Disconnecting...

[03:15:38] 🖥 Provisioning machine...

[03:15:38] 🎛 Preparing system...

[03:15:38] ⛓ Spinning up manager process...

[03:15:00] 🚀 Starting up repository: 'ann-abandono-clientes', branch: 'main', main module: 'app.py'

[03:15:00] 🐙 Cloning repository...

[03:15:01] 🐙 Cloning into '/mount/src/ann-abandono-clientes'...

[03:15:01] 🐙 Cloned repository!

[03:15:01] 🐙 Pulling code changes from Github...

[03:15:01] 📦 Processing dependencies...


──────────────────────────────────────── uv ───────────────────────────────────────────


Using uv pip install.

Using Python 3.12.10 environment at /home/adminuser/venv

Resolved 68 packages in 1.60s

Prepared 68 packages in 16.61s

Installed 68 packages in 1.08s

 + absl-py==2.2.2

 + altair==5.5.0

 + astunparse==1.6.3

 + attrs==25.3.0

 + blinker==1.9.0

 + cachetools==5.5.2

 + certifi==2025.4.26

 + charset-normalizer==3.4.2

 +[2025-05-13 03:15:21.651271]  click==8.2.0

 + flatbuffers==25.2.10

 + gast==0.6.0

 + gitdb==4.0.12

 + gitpython==3.1.44

 + google-pasta==0.2.0

 + grpcio==1.71.0

 + h5py[2025-05-13 03:15:21.651888] ==3.13.0

 + idna==3.10

 + jinja2==3.1.6

 + joblib==1.5.0

 + jsonschema[2025-05-13 03:15:21.652139] ==4.23.0

 + jsonschema-specifications==2025.4.1

 + keras==3.9.2

 + libclang==18.1.1[2025-05-13 03:15:21.652343] 

 + markdown==3.8

 + markdown-it-py==3.0.0

 + markupsafe==3.0.2

 +[2025-05-13 03:15:21.652589]  mdurl==0.1.2

 + ml-dtypes==0.5.1

 + namex==0.0.9

 + narwhals==1.39.0[2025-05-13 03:15:21.652806] 

 + numpy==2.1.3

 + opt-einsum==3.4.0

 + optree==0.15.0[2025-05-13 03:15:21.653011] 

 + packaging==24.2

 + pandas==2.2.3

 + pickle-mixin==1.0.2[2025-05-13 03:15:21.653500] 

 + pillow==11.2.1

 + protobuf==5.29.4

 + pyarrow[2025-05-13 03:15:21.653745] ==20.0.0

 + pydeck==0.9.1

 + pygments==2.19.1

 +[2025-05-13 03:15:21.653958]  python-dateutil==2.9.0.post0

 + pytz==2025.2

 + referencing==0.36.2

 + requests==2.32.3[2025-05-13 03:15:21.654154] 

 + rich==14.0.0

 + rpds-py==0.24.0

 + scikit-learn==1.6.1[2025-05-13 03:15:21.654350] 

 + scipy==1.15.3

 + setuptools==80.4.0

 + six==[2025-05-13 03:15:21.654560] 1.17.0

 + smmap==5.0.2

 + streamlit==1.45.1

 + tenacity==[2025-05-13 03:15:21.654768] 9.1.2

 + tensorboard==2.19.0

 + tensorboard-data-server==0.7.2

 + tensorflow==[2025-05-13 03:15:21.654979] 2.19.0

 + termcolor==3.1.0

 + threadpoolctl==3.6.0

 + toml==0.10.2

 + tornado==6.4.2[2025-05-13 03:15:21.655177] 

 + typing-extensions==4.13.2

 + tzdata==2025.2

 + urllib3==2.4.0

 + watchdog==6.0.0

 + werkzeug==3.1.3

 + wheel==0.45.1

 + wrapt==1.17.2

Checking if Streamlit is installed

Found Streamlit version 1.45.1 in the environment

Installing rich for an improved exception logging

Using uv pip install.

Using Python 3.12.10 environment at /home/adminuser/venv

Audited 1 package in 4ms


────────────────────────────────────────────────────────────────────────────────────────


[03:15:24] 🐍 Python dependencies were installed from /mount/src/ann-abandono-clientes/requirements.txt using uv.

Check if streamlit is installed

Streamlit is already installed

[03:15:26] 📦 Processed dependencies!




2025-05-13 03:15:54.062 Uncaught exception GET /_stcore/stream (127.0.0.1)

HTTPServerRequest(protocol='http', host='ann-abandono-clientes-hz6sxhgdblx6nxwukfehid.streamlit.app', method='GET', uri='/_stcore/stream', version='HTTP/1.1', remote_ip='127.0.0.1')

Traceback (most recent call last):

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/web/bootstrap.py", line 347, in run

    if asyncio.get_running_loop().is_running():

       ^^^^^^^^^^^^^^^^^^^^^^^^^^

RuntimeError: no running event loop


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/home/adminuser/venv/lib/python3.12/site-packages/tornado/websocket.py", line 938, in _accept_connection

    open_result = handler.open(*handler.open_args, **handler.open_kwargs)

                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/web/server/browser_websocket_handler.py", line 177, in open

    self._session_id = self._runtime.connect_session(

                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/runtime.py", line 384, in connect_session

    session_id = self._session_mgr.connect_session(

                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/websocket_session_manager.py", line 99, in connect_session

    session = AppSession(

              ^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/app_session.py", line 158, in __init__

    self.register_file_watchers()

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/app_session.py", line 195, in register_file_watchers

    self._local_sources_watcher = LocalSourcesWatcher(self._pages_manager)

                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/watcher/local_sources_watcher.py", line 65, in __init__

    self.update_watched_pages()

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/watcher/local_sources_watcher.py", line 77, in update_watched_pages

    self._register_watcher(

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/watcher/local_sources_watcher.py", line 136, in _register_watcher

    watcher=PathWatcher(filepath, self.on_file_changed),

            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/watcher/event_based_path_watcher.py", line 107, in __init__

    path_watcher.watch_path(

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/watcher/event_based_path_watcher.py", line 185, in watch_path

    folder_handler.watch = self._observer.schedule(

                           ^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/api.py", line 312, in schedule

    emitter.start()

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/utils/__init__.py", line 75, in start

    self.on_thread_start()

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/inotify.py", line 119, in on_thread_start

    self._inotify = InotifyBuffer(path, recursive=self.watch.is_recursive, event_mask=event_mask)

                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/inotify_buffer.py", line 30, in __init__

    self._inotify = Inotify(path, recursive=recursive, event_mask=event_mask)

                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/inotify_c.py", line 185, in __init__

    self._add_dir_watch(path, event_mask, recursive=recursive)

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/inotify_c.py", line 411, in _add_dir_watch

    self._add_watch(full_path, mask)

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/inotify_c.py", line 424, in _add_watch

    Inotify._raise_error()

  File "/home/adminuser/venv/lib/python3.12/site-packages/watchdog/observers/inotify_c.py", line 435, in _raise_error

    raise OSError(errno.ENOSPC, "inotify watch limit reached")

OSError: [Errno 28] inotify watch limit reached

Exception ignored in: <function AppSession.__del__ at 0x7fa3ed649800>

Traceback (most recent call last):

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/app_session.py", line 178, in __del__

    self.shutdown()

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/app_session.py", line 255, in shutdown

    self.request_script_stop()

  File "/home/adminuser/venv/lib/python3.12/site-packages/streamlit/runtime/app_session.py", line 428, in request_script_stop

    if self._scriptrunner is not None:

       ^^^^^^^^^^^^^^^^^^

AttributeError: 'AppSession' object has no attribute '_scriptrunner'

2025-05-13 03:16:01.228863: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.

2025-05-13 03:16:01.233900: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.

2025-05-13 03:16:01.243320: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:467] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered

WARNING: All log messages before absl::InitializeLog() is called are written to STDERR

E0000 00:00:1747106161.272816     211 cuda_dnn.cc:8579] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered

E0000 00:00:1747106161.281448     211 cuda_blas.cc:1407] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered

W0000 00:00:1747106161.308645     211 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

W0000 00:00:1747106161.308693     211 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

W0000 00:00:1747106161.308696     211 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

W0000 00:00:1747106161.308699     211 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.

2025-05-13 03:16:01.317301: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.

To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.

2025-05-13 03:16:14.035860: E external/local_xla/xla/stream_executor/cuda/cuda_platform.cc:51] failed call to cuInit: INTERNAL: CUDA error: Failed call to cuInit: UNKNOWN ERROR (303)

WARNING:absl:Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.

/home/adminuser/venv/lib/python3.12/site-packages/sklearn/base.py:380: InconsistentVersionWarning: Trying to unpickle estimator LabelEncoder from version 1.5.1 when using version 1.6.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:

https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations

  warnings.warn(

/home/adminuser/venv/lib/python3.12/site-packages/sklearn/base.py:380: InconsistentVersionWarning: Trying to unpickle estimator OneHotEncoder from version 1.5.1 when using version 1.6.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:

https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations

  warnings.warn(

/home/adminuser/venv/lib/python3.12/site-packages/sklearn/base.py:380: InconsistentVersionWarning: Trying to unpickle estimator StandardScaler from version 1.5.1 when using version 1.6.1. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:

https://scikit-learn.org/stable/model_persistence.html#security-maintainability-limitations

  warnings.warn(

1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 61ms/step[2025-05-13 03:16:14.704888]
